content/_partials/mailing-lists/jenkins-infra.adoc

```diff
@@ -1,4 +1,4 @@
-Mailing list for jenkins-ci.org operations and infrastructure.
+Mailing list for jenkins.io operations and https://www.jenkins.io/projects/infrastructure/[Jenkins Infrastructure].
```

Comment:
We use site local links so that development instances of the pages do not surprise contributors by linking back to the releases site.
```suggestion
Mailing list for jenkins.io operations and link:/projects/infrastructure/[Jenkins Infrastructure].
```

Reply:
Missed that, thanks!